python/paddle/tensor/creation.py

```diff
@@ -3379,3 +3379,75 @@ def set_(
             shape = source.shape
 
         return _C_ops.set_(x, source, shape, stride, offset)
+
+
+@inplace_apis_in_dygraph_only
+def resize_(
+    x: paddle.Tensor, shape: Sequence[int], fill_zero: bool = False
+) -> paddle.Tensor:
+    """
+    Resize ``x`` with specified ``shape``.
+
+    Args:
+        x (Tensor): An arbitrary Tensor. The data type supports ``bfloat16``, ``float16``, ``float32``, ``float64``,
+            ``bool``, ``int8``, ``int16``, ``int32``, ``int64``, ``uint8``, ``complex64`` or ``complex128``.
+        shape (list|tuple, optional): Define the target shape. Each element of it should be integer.
+        fill_zero (bool, optional): If the size of specified ``shape`` is greater than the original Tensor size, the
+            new Tensor will be filled with zero if ``fill_zero`` is True. Default: False, which means the filled value
+            will be undetermined.
+        name (str|None, optional): Name for the operation (optional, default is None). For more information, please refer to :ref:`api_guide_Name`.
```

Comment:
Where did this parameter come from?

Reply:
To make the functionality aligned with numpy (when calling `ndarray.resize()` )

![image](https://github.com/user-attachments/assets/e84c5238-555c-4219-a563-21cefc1a25eb)

Reply:
Then you should add this parameter to interface?

Reply:
> Then you should add this parameter to interface?

Do you mean I should not add this parameter and functionality to the interface?

Reply:
your interface have three inputs, but doc get four?